geth/common/types.go

```diff
@@ -376,10 +376,11 @@ func (e APIError) Error() string {
 
 // AccountInfo represents account's info
 type AccountInfo struct {
-	Address  string `json:"address"`
-	PubKey   string `json:"pubkey"`
-	Mnemonic string `json:"mnemonic"`
-	Error    string `json:"error"`
+	Address    string `json:"address"`
```

Comment:
As mentioned above: While adding the error as string for marshalling is fine the original error has to be a return value of the function.